packages/react-charting/src/components/Legends/Legends.types.ts

```diff
@@ -238,6 +238,22 @@ export interface ILegendsProps {
    */
   defaultSelectedLegend?: string;
 
+  /**
+   * Keys (title) that will be used to set selected items.
+   * Specifying this prop sets the component to controlled mode when `canSelectMultipleLegends` is `true`.
+   * @see defaultSelectedLegends for setting the initially-selected legends in uncontrolled mode.
+   * @see selectedLegend for setting the selected legend when `canSelectMultipleLegends` is `false`.
+   */
+  selectedLegends?: string[];
```

Comment:
Could you add a comment on how selectedLegends and defaultSelectedLegends are different.

Reply:
I've extended the descriptions for both the new properties and the existing `default*` properties to highlight the behavior differences.  I'm not certain this will render nicely in the docsite, however &mdash; I can attempt to make it more concise if necessary.